### PR TITLE
fix(download): download failed when using ELECTRON_MIRROR

### PR DESCRIPTION
**Pull request by bytemain**
this close: #73

### CODE REVIEW

File: pkg/electron/electronDownloader.go [hunk 1]
```diff
@@ -95,21 +95,33 @@ func getBaseUrl(config *ElectronDownloadOptions) string {
...
 }
 
+func getVersionWithPrefixV(version string) string {
+	tmp := version
+	if !strings.HasPrefix(tmp, "v") {
+		tmp = "v" + tmp
+	}
+	return tmp
+}
+
```
Comment (BlackHole1) [inline pkg/electron/electronDownloader.go:117]:
```suggestion
func normalizeVersion(version string) string {
	if strings.HasPrefix(version, "v") {
		return version
	}
	return "v" + version
}

```

File: pkg/electron/electronDownloader.go [hunk 2]
```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/develar/app-builder/pkg/log"
 	"github.com/develar/app-builder/pkg/util"
 	"github.com/develar/errors"
-	"github.com/develar/go-fs-util"
-	"github.com/json-iterator/go"
+	fsutil "github.com/develar/go-fs-util"
+	jsoniter "github.com/json-iterator/go"
```
Comment (BlackHole1) [inline pkg/electron/electronDownloader.go:14]: Superfluous changes?

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Comment by BlackHole1** [comment]
title nit: `fix(download): download failed when using ELECTRON_MIRROR`

File: pkg/electron/electronDownloader.go (hunk 2 above)
Comment (lengthmin) [reply pkg/electron/electronDownloader.go:14]: I think this is added by `go fmt`, so should I remove this?

**Comment by lengthmin** [comment]
@develar please have a look

**Review by BlackHole1 — APPROVED** [review]
LGTM

cc: @develar

**Comment by fengmk2** [comment]
+1

**Comment by BlackHole1** [comment]
I am trying to get in touch with @develar

**Comment by BlackHole1** [comment]
Hey @develar. Can a new version be released for this purpose? If a new version of this is released, I will submit a PR to electron-builder for upgrading this library

**Comment by develar** [comment]
`4.1.2` is released. Sorry for the delay. Thanks for your help.

**Pull request merged**